.github/workflows/test.yml

```diff
@@ -126,8 +126,12 @@ jobs:
           CODECOV_BASH_SHA512SUM: d075b412a362a9a2b7aedfec3b8b9a9a927b3b99e98c7c15a2b76ef09862aeb005e91d76a5fd71b511141496d0fd23d1b42095f722ebcd509d768fba030f159e
           CODECOV_TOKEN: ${{ secrets.CODECOV_TOKEN }}
         run: |
+          if [[ "${{ matrix.platform }}" == macos* ]]; then
+            shopt -s expand_aliases
+            alias sha512sum='shasum -a 512'
+          fi
           curl -fsSLO "https://raw.githubusercontent.com/codecov/codecov-bash/${CODECOV_BASH_VERSION}/codecov"
-          echo "$CODECOV_BASH_SHA512SUM codecov" | sha512sum -c -
+          echo "$CODECOV_BASH_SHA512SUM  codecov" | sha512sum -c -
```

Comment:
For some reason (I didn't fully get), without the extra space it doesn't work on MacOS, but with it, it works correctly in all platforms 🤷🏻

Reply:
I remember that we had issues with `codecov` (which later were resolved) and created [the task](https://github.com/grafana/k6/issues/2797), but one of the highlights was that `codecov` has the GH action. So maybe it's worth migrating to it

Reply:
Oh, nice! Thanks for letting me know, I could consider working on that right after, but I'd prefer to keep it for a separate PR, if that sounds good to you as well 🙏🏻